app/0.14.x/cli.md

```diff
@@ -213,4 +213,158 @@ Options:
 
 [Back to TOC](#table-of-contents)
 
+---
+
+## Stopping Kong gracefully
+
+Stopping kong can be achieved with a simple `kong stop` command. Yet, depending
+on the environment Kong is running in, a more controlled shutdown might be
+desired. To understand the differences in the many ways you can shut down Kong
+we need to have a look at how the mechanism works.
+
+The related Kong CLI commands operate by sending Unix signals to the Kong
+master process (generally there is no need to interact with the worker processes
+in this way). The signals used are:
+
+  - `SIGTERM` will gracefully exit the master process, but master will in turn
+    forcefully shut down the workers. So workers will abruptly close any open
+    connections and exit. Yet master will properly clean up (PID files etc.).
+  - `SIGQUIT` for a graceful shutdown of both the workers and master, the
+    workers will finish their active connections and timers before exiting.
+    Note: while exiting the workers will no longer accept new connections.
+  - `SIGKILL` will forcefully exit the master process (workers may remain). None
+    of the kong CLI commands will actually use this signal.
+
+The Kong CLI commands will:
+
+  - `[stop](#kong-stop)` will send `SIGTERM`, and hence will be forcefully
```

Comment:
This renders incorrectly:

![cli_reference_-_v0_14_x___kong_-_open-source_api_management_and_microservice_management](https://user-images.githubusercontent.com/94904/48432165-7b693280-e728-11e8-8731-e09f176ffe2a.jpg)


The correct code to render monospaced text as a link is:

`[`stop`](#kong-stop)`

Reply:
Just FYI, you don't even have to run the docs site locally anymore to see issues like this - we've now got automatic Netlify-powered previews! See https://github.com/Kong/docs.konghq.com/pull/1040 - very handy!

Reply:
I just noticed, awesome!

Reply:
@coopr I clicked on #1040 twice before I realized I was in a recursive loop :) what's the correct link with the Netlify goodness? I'm curious about this!

Reply:
Nevermind, saw it!

Reply:
You'll see the Netlify preview notifications in the comments thread - here is one https://github.com/Kong/docs.konghq.com/pull/1040#issuecomment-438338344 and at the bottom of the thread in the "checks" section you'll see another.